src/encoded/static/components/navigation.js

```diff
@@ -86,11 +86,56 @@ export default class Navigation extends React.Component {
      * @param {object} e React synthetic event
      */
     dropdownClick(dropdownId, e) {
-        // After clicking the dropdown trigger button, don't allow the event to bubble to the rest of the DOM.
-        e.nativeEvent.stopImmediatePropagation();
-        this.setState(prevState => ({
-            openDropdown: dropdownId !== prevState.openDropdown ? dropdownId : '',
-        }));
+        // Check if device has touch events
+        function isMobile() {
+            try {
+                document.createEvent('TouchEvent');
+                return true;
+            } catch (evt) {
+                return false;
+            }
+        }
+        const isMobileDevice = isMobile();
```

Comment:
I think this should be added to browserfeat.js so that any code can tell touch events are supported, and CSS can target it. But not necessarily in this ticket. [Modernizr](https://modernizr.com/docs/) has a "touchevents"/"no-touchevents" CSS class.